doc/conf.py

```diff
@@ -119,9 +117,7 @@ def usermethod():
         except Exception as e:
             docname = self.state.document.settings.env.docname
             raise self.error(
-                "Unable to execute embedded doc code at {}:{} ... {}\n{}".format(
-                    docname, self.lineno, datetime.datetime.now(), str(e)
-                )
+                f"Unable to execute embedded doc code at {docname}:{self.lineno}\n{str(e)}"
```

Comment:
I found the timestamp really annoying and not useful when trying to debug a docs build. But you can certainly reject the edit if you disagree!